src/material/chips/chip-input.ts

```diff
@@ -75,16 +75,19 @@ export class MatChipInput implements MatChipTextControl, AfterContentInit, OnCha
 
   /** Whether the control is focused. */
   focused: boolean = false;
-  _chipGrid: MatChipGrid;
 
   /** Register input for chip list */
   @Input('matChipInputFor')
+  get chipGrid(): MatChipGrid {
+    return this._chipGrid;
+  }
   set chipGrid(value: MatChipGrid) {
     if (value) {
       this._chipGrid = value;
       this._chipGrid.registerInput(this);
     }
   }
+  _chipGrid: MatChipGrid;
```

Comment:
Should we make this private now that there's a getter for it?

Reply:
Sure!

Reply:
Note: In this PR, we expose chipGrid by using getter function and make _chipGrid property private. This may break your build if you use _chipGrid in your code, change it to chipGrid will fix the issue.

Reply:
This is ok, `_chipGrid` was an internal property to begin with, so we do not consider this a breaking change